packages-native/bottom-sheet/src/components/NativeBottomSheet.tsx

```diff
@@ -32,10 +33,10 @@ export const NativeBottomSheet = (props: NativeBottomSheetProps): ReactElement =
 
     const actionHandler = useCallback(
         (index: number) => {
-            const action = props.itemsBasic[index].action;
-            if (action && action.canExecute) {
-                action.execute();
-            }
+            setTimeout(() => {
+                executeAction(props.itemsBasic[index].action);
```

Comment:
Shouldn't you add `props.itemsBasic` to your `useCallback` dependencies?

Reply:
Isn't it already there https://github.com/mendix/widgets-resources/blob/4680765147e214b21aea769eba1da6393d2e94b6/packages-native/bottom-sheet/src/components/NativeBottomSheet.tsx#L45 ? 😉